validation/open_boundaries/cylinder.jl

```diff
@@ -0,0 +1,130 @@
+# This validation script shows open boundaries working in a simple case where the
+# flow remains largely unidirectional and so at one end we have no matching scheme
+# but just prescribe the inflow. At the other end we then make no assumptions about
+# the flow and use a very simple open boundary condition to permit information to 
+# exit the domain. If, for example, the flow at the prescribed boundary was reversed
+# then the model would likely fail.
+
+using Oceananigans, Adapt, CairoMakie
+using Oceananigans.BoundaryConditions: FlatExtrapolationOpenBoundaryCondition
+
+import Adapt: adapt_structure
+
+@kwdef struct Cylinder{FT}
+    D :: FT = 1.
+   x₀ :: FT = 0.
+   y₀ :: FT = 0.
```

Comment:
i think this style is against YASGuide, we have to use 0.0 and 1.0

Reply:
please check

Reply:
Oh yeah I think I wrote this a while ago sorry